hazelcast-spring/src/main/java/com/hazelcast/spring/HazelcastConfigBeanDefinitionParser.java

```diff
@@ -501,7 +532,7 @@ public void handleTcpIp(Node node, BeanDefinitionBuilder joinConfigBuilder) {
                             "interface", "member", "members");
             final ManagedList members = new ManagedList();
             for (Node n : childElements(node)) {
-                String name = xmlToJavaName(cleanNodeName(n));
+                String name = cleanNodeName(n);
```

Comment:
that was a bug. here the name is compared with xsd attribute name, not with its java representation. have a look at other similar places.